SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Security Policy 
+ 
+## Security Contacts and Procedures
+ 
+The narayana community takes security seriously and will take immediate action to address security-related problems that involve our products or services.
+ 
+Please report any suspected security vulnerability in this project to Red Hat Product Security at secalert@redhat.com. You can use our GPG key to communicate with us securely.
+ 
+To report an issue in any Red Hat branded website or online service, please contact Red Hat Information Security at site-security@redhat.com.
+https://access.redhat.com/security/team/contact
```

Comment:
Maybe, we can add a link to the page where this wording was taken from? i.e. https://access.redhat.com/security/team/contact ? I thought to do so as info like `our GPG key` (and especially *our* -> Narayana community doesn't have a GPG key to communicate with us :-) ) can be a bit tricky to find.